docs/self-managed/installation-methods/helm/upgrade/helm-870-880.md

```diff
@@ -185,7 +185,38 @@ The following entities will be migrated:
 
 Once the migration is complete, you can safely delete the job, its associated ConfigMap, and the migration client in Management Identity.
 
-### 5. Run the Upgrade
+### 5. Process Migration
+
+The Process Migration app migrates from the old, separated indices in Elasticsearch or OpenSearch on 8.7 to the new unified indices on 8.8.
+
+In 8.7 and previous versions there were two indices:
+
+- `tasklist-process`
+- `operate-process`
+
+From 8.8 onwards, there will only be one `operate-process` index for the needs of the entire Camunda Cluster.
+
+:::warning
+
+- The process migration can't be ran on a fresh install of 8.8. The migration will only be successful on upgrade from 8.7.
+- The migration can only run once.
+- Before upgrading, the 8.7 zeebe workers should already have some load and should NOT be idle. Otherwise the migration will fail.
+  :::
+
+To enable the process migration, the following value can be modified:
+
+```
+core:
+  migration:
+    process:
+      enabled: true
+```
+
+By default, the tasklist, and operate importers are disabled in 8.8. When the above value is set to true, the tasklist and operate importers are eanbled. This is because the migration will wait for the importers to stop processing data. If the importers do not exist then the migration will not complete successfully.
```

Comment:
"If the importers do not exist then the migration will not complete successfully."
Do we mean here that if this option is not enabled, the exporters will not be active and migration will not happen?

Reply:
@panagiotisgts I would appreciate it if you could explain further for @theburi 
I will resolve for now since I don't think a change is needed here.